spec/rails_helper.rb

```diff
@@ -147,6 +147,12 @@
     stub_env_variables_for_gai
   end
 
+  config.before(:each, type: :view) do
+    allow(ActionView::Base)
+      .to receive(:default_form_builder)
+          .and_return GOVUKDesignSystemFormBuilder::FormBuilder
```

Comment:
Needed to fix view specs defaulting to the regular rails form builder since they are outside the scope of `ApplicationController` - where we are setting the default form builder